Tests/SwiftParserTest/translated/TrailingSemiTests.swift

```diff
@@ -68,7 +77,21 @@ final class TrailingSemiTests: XCTestCase {
         DiagnosticSpec(locationMarker: "2️⃣", message: "consecutive declarations on a line must be separated by ';'", fixIts: ["insert ';'"]),
         DiagnosticSpec(locationMarker: "3️⃣", message: "consecutive declarations on a line must be separated by ';'", fixIts: ["insert ';'"]),
         DiagnosticSpec(locationMarker: "4️⃣", message: "consecutive declarations on a line must be separated by ';'", fixIts: ["insert ';'"]),
```

Comment:
These errors just got me thinking that we should also be offering to add a newline instead of a semicolon. That’s much more likely what the developer wants than the semicolon.

Reply:
https://github.com/apple/swift-syntax/issues/1612